src/test/MockPolygonBridge.t.sol

```diff
@@ -17,7 +17,7 @@ contract MockPolygonBridgeTest is PRBTest, StdCheats {
     event RootHistoryExpirySet(uint256 rootHistoryExpiry);
 
     /// @notice Thrown when new root is inserted
```

Comment:
Nit: consider changing `Thrown` to `Emitted` which is typically seen more often in Solidity. Not something that is needed now, but just a thought to keep in mind for the future.

Reply:
done